lib/runCode.js

```diff
@@ -193,6 +193,7 @@ module.exports = function (opts, cb) {
         // run the opcode
         var result = opFn.apply(null, args)
       } catch (e) {
+				runState.vmError = e
```

Comment:
any error occurred here does not belong to any of the known exceptions, so excluding them from the runState will break stateroots

Reply:
I see @axic has incorporated a new type of error `INTERNAL_ERROR` (https://github.com/ethereumjs/ethereumjs-vm/blob/35e8ddcc71ae47954d5a2599d784b3e63507df6f/lib/constants.js#L9) in https://github.com/ethereumjs/ethereumjs-vm/pull/222 .

Maybe we can pull those changes to this PR, and add a new field that allows us to specify a custom error message for `INTERNAL_ERROR`s .